elfo-core/src/request_table.rs

```diff
@@ -251,7 +251,7 @@ impl ResponseToken {
 
     #[doc(hidden)]
     #[inline]
-    pub fn forget(mut self) {
+    pub fn forget(&mut self) {
```

Comment:
Is there a use for `RequestToken` after forgetting it?

Reply:
Actually, no. It's used only when sending to another node (semantically, a token is moved) and in `Envelope::drop_as_unused.` The second usage requires leaving `Envelope` valid (in order to run `impl Drop`), but it can be replaced with `mem::replace(header.kind, MessageKind::Regular { .. })` instead

Reply:
If `.forget()` is a part of the public API (which it seems not to be, based on `#[doc(hidden)]`), I think it should take `self` and `impl Drop` should use another method